src/streamlink/plugins/eltrecetv.py

```diff
@@ -24,16 +24,16 @@ def _get_streams(self):
             http.headers = {'Referer': self.url,
             'User-Agent': useragents.CHROME}
             res = http.get(self.url)
-            video_search = res.text
-            video_search = video_search[video_search.index('data-kaltura="') +14:]
-            video_search = video_search[: video_search.index('"')]
-            video_search = video_search.replace("&quot;", '"')
-            json_video_search = parse_json(video_search)
+            _player_re = re.compile(r'''data\-kaltura="([^"]+)"''')
```

Comment:
You are right, i just copy and paste the @back-to regex suggestion :stuck_out_tongue: